core-plugins/docs/OracleExport-action.md

```diff
@@ -0,0 +1,94 @@
+# Oracle Export Action
+
+
+Description
+-----------
+A Hydrator Action plugin to efficiently export data from Oracle to HDFS or local file system. The plugin uses
+Oracle's command line tools to export data. The data exported from this tool can then be used in Hydrator pipelines.
+
+
+Use Case
+--------
+A Hydrator user would like to export oracle data onto HDFS or local file system using an action plugin
+that exports Oracle data using the Oracle bulk export tool as opposed to using JDBC.
+
+
+Properties
+----------
+
+**oracleServerHostname:** Hostname of the remote Oracle Host.
```

Comment:
@abhinavbansal246  'commandToRun will be executed on the host machine' is already added in the descriptions. Please review.